app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -134,31 +134,31 @@ def parse_node(node)
       new_result.merge!(
         :type                       => 'ManageIQ::Providers::Kubernetes::ContainerManager::ContainerNode',
         :identity_infra             => node.spec.externalID,
-        :identity_machine           => node.status.nodeInfo.machineID,
-        :identity_system            => node.status.nodeInfo.systemUUID,
-        :container_runtime_version  => node.status.nodeInfo.containerRuntimeVersion,
-        :kubernetes_proxy_version   => node.status.nodeInfo.kubeProxyVersion,
-        :kubernetes_kubelet_version => node.status.nodeInfo.kubeletVersion,
+        :identity_machine           => node.status.try(:nodeInfo).try(:machineID),
```

Comment:
if the first run for node.status.try(:nodeInfo) returns nil, then lines 138-141 are essentially irrelevant, no?